src/modal/__tests__/modal.test.tsx

```diff
@@ -140,21 +140,6 @@ describe('Modal component', () => {
       const wrapper = renderModal({ visible: true });
       expect(document.activeElement).toBe(wrapper.findDismissButton().getElement());
     });
-
-    it('restores focus to previous element on unmount', () => {
```

Comment:
The same is tested in the integ test "should reset focus to previously active element". Having the unit test is problematic because the react-focus-lock returnFocus implementation seem to not be supported in jsdom.